modin/experimental/cloud/rayscale.py

```diff
@@ -136,23 +136,30 @@ def __make_config(self):
 
         # NOTE: setup_commands may be list with several sets of shell commands
         # this change only first set defining the remote environment
-        res = self._update_conda_requirements(config["setup_commands"][0])
+        res = self._update_conda_requirements(
+            config["setup_commands"][0], self.add_conda_packages
+        )
         config["setup_commands"][0] = res
 
         return _bootstrap_config(config)
 
     @classmethod
-    def _conda_requirements(cls):
+    def _conda_requirements(cls, add_conda_packages: list = None):
```

Comment:
this should not be a `@classmethod`, then you can stop passing `self.add_conda_packages` as parameter and just use it inside :)